src/mixed-line-bar-chart/format-highlighted.ts

```diff
@@ -56,11 +71,40 @@ function getSeriesDetail<T>(internalSeries: InternalChartSeries<T>, targetX: T):
   if (isDataSeries(series)) {
     for (const datum of series.data) {
       if (matchesX(targetX, datum.x)) {
+        const customContent = detailPopoverSeriesContent
+          ? detailPopoverSeriesContent({ series, x: targetX, y: datum.y })
+          : undefined;
+        const hasSubItems = !!customContent?.subItems?.length;
+        const isExpandable = customContent?.expandable && hasSubItems;
+        const isKeyString = typeof customContent?.key === 'string';
+        const key = customContent?.key && (!isExpandable || isKeyString) ? customContent.key : series.title;
```

Comment:
Can we just use the series title? What is the point in trying for the content key first?

Reply:
But what do we lose if we just use the series.title?

Reply:
This is passed to render the popover details, so we would lose the ability to render links in the key of the key-value pairs. Similar with `customContent.value` which contains the ReactNode value if `detailPopoverSeriesContent` was provided.

Reply:
ah, got it now - thanks!